src/main/webapp/js/science_portal.js

```diff
@@ -200,32 +200,29 @@
 
 
           // $itemContainer holds both the linkItem with the
-          // connect and delete controls, sesion type logo, etc.,
+          // connect and delete controls, session type logo, etc.,
           // and potentially a 'blockingDiv' that puts a panel
           // over the linkItem, blocking access when the session
           // isn't Running.
           var $itemContainer = $('<div />')
           $itemContainer.prop('class', 'sp-link-container')
 
+          // Create the main $linkItem div to hold session
+          // information and action controls
+          var $linkItem = $('<div />')
+          $linkItem.prop('class', 'sp-link-connect')
```

Comment:
The `sp-link-connect` class was removed from the CSS above but used here.  Is this a typo or meant to be an identifying class?

Reply:
Turns out the whole div wasn't necessary. :)